<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDateDialog</class>
 <widget class="QDialog" name="SetDateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>220</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit date</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>197</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vertical_layout">
    <item>
     <widget class="QDateEdit" name="date_edit">
      <property name="displayFormat">
       <string>MM.dd.yyyy</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizonta_layout">
      <item>
       <widget class="QPushButton" name="set_button">
        <property name="text">
         <string>Set new date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_button">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
